src/main/java/io/github/moulberry/notenoughupdates/events/RepositoryReloadEvent.java

```diff
@@ -23,7 +23,7 @@
 
 public class RepositoryReloadEvent extends NEUEvent {
 	private final File baseFile;
-	private boolean isFirstLoad;
```

Comment:
Important lol

Reply:
I don't get this comment. IntelliJ just suggested adding final, since we don't change the value anywhere. You want me to delete the field completely, or want me to keep it non-final?

Reply:
nah, its fine, just kind of out of nowhere with no other changes